### ISSUE TITLE
update dependences,got error

### ISSUE THREAD

**Issue by leanhunter** [start-react/native-starter-kit#18]
```
    "native-base": "0.5.0",
    "react": "15.2.0",
    "react-native": "0.29.2",
```

got error  running  react-native run-ios

**Comment by leanhunter** [comment]
"native-base": "0.5.0",
        "react": "15.2.0",
        "react-native": "0.29.2",


got error  running  react-native run-ios

**Comment by SupriyaKalghatgi** [comment]
@leanhunter Yes, soon Native Starter Kit will have updated versions of [React](https://facebook.github.io/react/), [React Native](https://facebook.github.io/react-native/) and [NativeBase](http://nativebase.io/).

**Comment by leanhunter** [comment]
@SupriyaKalghatgi  tks~

**Comment by SupriyaKalghatgi** [comment]
@leanhunter Fixed with [v5.0.0](https://github.com/start-react/native-starter-kit/releases/tag/v5.0.0)
